### ISSUE TITLE
`MP_SMTP_AUTH_ALLOW_INSECURE` can not be configured via env variable

### ISSUE THREAD

**Issue by IgnisDa** [axllent/mailpit#77]
I set `MP_SMTP_AUTH_ALLOW_INSECURE = 1` in my environment.

I get this error.

```
ERRO[2023/03/31 16:13:13] SMTP authentication requires TLS encryption, run with `--smtp-auth-allow-insecure` to allow insecure authentication
```

I think it means that the enviroment variable is not being recognized.

Same applies to the `MP_SMTP_AUTH_ACCEPT_ANY` variable.

**Comment by axllent** [comment]
Can you please cop & paste the exact command you are running to start Mailpit, including trying to set the environment?

**Comment by IgnisDa** [comment]
I have this in my `docker-compose.yml`.

```yml

  mail:
    image: axllent/mailpit
    ports:
      - 8025:8025
    environment:
      MP_DATA_FILE: /data/mail.data
      MP_SMTP_AUTH_ACCEPT_ANY: '1'
      MP_SMTP_AUTH_ALLOW_INSECURE: '1'
    healthcheck:
      test: [ "CMD", "wget", "--no-verbose", "--tries=1", "--spider", "http://localhost:8025/readyz" ]
      interval: 5s
      timeout: 5s
      retries: 3
    volumes:
      - mail-data:/data
```

**Comment by IgnisDa** [comment]
Looking at https://github.com/axllent/mailpit/blob/321c181a9c5e855e990ed1fe7e1305e1ad456cc0/cmd/root.go#L241-L248, it should be working fine. I wonder why it does not.

**Comment by axllent** [comment]
I would say because you are using `'` in your config, so the value is seen literallt as `'1'` instead of `1`. From memory you can only use `""` or none at all. Try:
```yaml
      MP_SMTP_AUTH_ACCEPT_ANY: 1
      MP_SMTP_AUTH_ALLOW_INSECURE: 1
```
or
```yaml
      MP_SMTP_AUTH_ACCEPT_ANY: "1"
      MP_SMTP_AUTH_ALLOW_INSECURE: "1"
```

**Comment by IgnisDa** [comment]
Tried both, did not work.

**Comment by axllent** [comment]
You are absolutely correct, these flags weren't being recognised, my apologies. I found the bug and have just released v1.5.3 which will fix it for you. The latest docker image has been updated, so make sure to pull the latest (`docker pull axllent/mailpit`)

**Comment by IgnisDa** [comment]
Thanks!
